src/main/io/displayport_msp_bf_compat.c

```diff
@@ -293,34 +294,35 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_THR:
         return BF_SYM_THR;
 
-/*
     case SYM_TEMP_F:
-        return BF_SYM_TEMP_F;
+        return BF_SYM_TEMPERATURE;
 
     case SYM_TEMP_C:
-        return BF_SYM_TEMP_C;
-*/
+        return BF_SYM_TEMPERATURE;
+
```

Comment:
There are actual temperature symbols for °F and °C in the betaflight font. I think you just need to remove the word `_TEMP` from the above 2 and it will work. For example `BF_SYM_F`. A search in the code should find these.